controllers/nodeagent_test.go

```diff
@@ -95,11 +95,12 @@ func TestDPAReconciler_buildNodeAgentDaemonset(t *testing.T) {
 		},
 	}
 	tests := []struct {
-		name    string
-		fields  fields
-		args    args
-		want    *appsv1.DaemonSet
-		wantErr bool
+		name          string
+		fields        fields
```

Comment:
lets remove `fields`, it not necessary. Only one test sets it, but removing it, did not break it. On test run, we just need client for dpaReconciler, which now use fakeClient

Reply:
Done, updated